docs/application/dotnet/samples/appfw/overview.md

```diff
@@ -39,23 +55,81 @@
+			<a href="https://developer.tizen.org/development/sample/native/AppFW/Badges" target="_blank">Native version</a></p>
+			</td>
+		</tr>
+		<tr>
+			<td><img alt="" height="180" src="media/wbadges.png" width="180"/></td>
+			<td>
+			<p><strong>(W) Badges</strong> [In progress]</p>
+			<p>This sample application demonstrates how to use application badge.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="180" src="media/wbadgecounter.png" width="180"/></td>
 			<td>
 			<p><strong>(W) BadgeCounter</strong> [In progress]</p>
-			<p>BadgeCounter is a sample application that demonstrates how to manage badge counter of the application using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.html" target="_blank">Tizen.Applications C# API</a>.</p>
+			<p>This sample application demonstrates how to manage badge counter of the application using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.html" target="_blank">Tizen.Applications C# API</a>.</p>
+			<p>In addition, there is similar web sample application.<br>
+			<a href="https://developer.tizen.org/development/sample/web/Application/Badges" target="_blank">Web version</a></p>
 			</td>
-		</tr>		
+		</tr>
+		<tr>
+			<td><img alt="" height="180" src="media/wmessageport.png" width="180"/></td>
+			<td>
+			<p><strong>(W) MessagePortSampleApp</strong> [In progress]</p>
+			<p>This sample application demonstrates how to send and receive messages between applications.</p>
+			<p>
+			<ul>
+				<li>Send a message to Tizen .Net Service sample application when <b>Send a message</b> button is clicked.</li>
+				<li>Receive a message from <a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/ServiceApp" target="_blank">Tizen .Net Service sample application</a></li>
+			</ul></p>
+			<p><b>Prerequisites</b><br>
+			First of all, you need to install and execute <a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/ServiceApp" target="_blank">Tizen .Net Service sample application</a> because it is used to communicate each other.<br>
+			You can install it by using Visual Studio or the sdb command line as follows:<br>
+			<code>$ sdb install org.tizen.example.ServiceApp-1.0.0.tpk</code><br>
+			<code>$ sdb shell</code><br>
+			<code>$ app_launcher -s org.tizen.example.ServiceApp
+			</code><br>
+			If you do not install Tizen .Net service application, an exception occurs when you press <b>Send a message</b> button.
+			</p>
+			<p>This application uses Tizen.Application.Messages API.<br>
+			<ul>
+				<li><a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.Messages.MessagePort.html" target="_blank">Class MessagePort</a></li>
+			</ul></p>
+			<p>In addition, there is similar native sample application.<br>
+			<a href="https://developer.tizen.org/development/sample/native/AppFW/%28Tutorial%29_Message_Port" target="_blank">Native version</a></p>
```

Comment:
If listing is mandatory above then, to maintain consistency, please follow the same style here as well. Otherwise it looks odd.
Please verify and share your opinion.

Reply:
I think it's good to maintain a format similar to that of other apps.

Reply:
OK. Then please follow the style and maintain the consistency.